flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -112,7 +113,12 @@ class DataStreamOverAggregate(
               "condition.")
         }
       case _: RowTimeType =>
-        throw new TableException("OVER Window of the EventTime type is not currently supported.")
+        if (overWindow.lowerBound.isUnbounded && overWindow.upperBound.isCurrentRow) {
+          createUnboundedAndCurrentRowEventTimeOverWindow(inputDS)
```

Comment:
Oh, I just recalled why I hadn't added an event-time OVER ROWS windows. They cannot deal with late data because all results would need to be shifted if a late record would be inserted. 

I would propose to keep this implementation anyway, and just throw an exception if a user enables handling of late data. What do you think @hongyuhong and @sunjincheng121?